samples/snippets/storage_download_into_memory.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
nit: update to Copyright 2021

Reply:
Same for the upload sample, thanks! Whoa I'm not sure if it's Google Inc or Google LLC. Let me check.

Reply:
it seems the 2021 ones have Google LLC. updated

Reply:
yep thanks, i just checked OSPO, and it's Google LLC